.travis.yml

```diff
@@ -2,10 +2,8 @@ language: go
 go:
   - tip
 before_script:
-  - go get golang.org/x/tools/cmd/vet
   - go get golang.org/x/tools/cmd/goimports
   - go get -u github.com/golang/lint/golint
-  - go get golang.org/x/tools/cmd/cover
```

Comment:
I don't think so yet; I'm fairly sure this was pulled from the CFSSL travis.yml.